examples/react/basic/package.json

```diff
@@ -12,23 +12,23 @@
     "format": "prettier --write . '!src/gen' && eslint . --fix && license-header"
   },
   "dependencies": {
-    "@bufbuild/buf": "1.43.0",
-    "@bufbuild/protobuf": "^2.2.0",
-    "@bufbuild/protoc-gen-es": "^2.2.0",
-    "@connectrpc/connect": "^2.0.0-rc.1",
+    "@bufbuild/buf": "1.46.0",
+    "@bufbuild/protobuf": "^2.2.1",
+    "@bufbuild/protoc-gen-es": "^2.2.1",
+    "@connectrpc/connect": "^2.0.0-rc.2",
     "@connectrpc/connect-query": "workspace:*",
-    "@connectrpc/connect-web": "^2.0.0-rc.1",
+    "@connectrpc/connect-web": "^2.0.0-rc.2",
     "@connectrpc/protoc-gen-connect-query": "workspace:*",
-    "@tanstack/react-query": "^5.56.2",
-    "@tanstack/react-query-devtools": "^5.58.0",
-    "@testing-library/jest-dom": "^6.5.0",
+    "@tanstack/react-query": "^5.59.16",
```

Comment:
This causes CI to fail. 

A react query API changed, which was addressed in https://github.com/connectrpc/connect-query-es/pull/469, but only partially. The same change needs to be applied to createInfiniteQueryOptions.